### ISSUE TITLE
Why there's no gradients required in proposal_target_layer?

### ISSUE THREAD

**Issue by Jossome** [sshaoshuai/PointRCNN#153]
The line is here: https://github.com/sshaoshuai/PointRCNN/blob/master/lib/net/rcnn_net.py#L123
I tried to remove the `with torch.no_grad():`, and there's a lot of inplace operations in it's forward method where gradient computation fails.

Could you explain why gradient is not necessary here? And how can I make the gradients be calculated properly? Thanks.
